onnxruntime/core/optimizer/constant_folding.cc

```diff
@@ -103,7 +103,19 @@ Status ConstantFolding::ApplyImpl(Graph& graph, bool& modified, int graph_level,
         continue;
       }
 
-      // Create execution frame for executing constant nodes.
+      // Constant folding doesn't support external initializers.
```

Comment:
It's failing to load the external initializer because the base model path isn't being passed through, so it can't create the full path to the external data in TensorProtoToMLValue. Would it be better to fix that instead of disabling constant folding?

The model path is available from Graph::ModelPath.

Reply:
Thanks @skottmckay ! Just pushed another fix following your advice.